plugins/kn-admin/cmd/domain/domain.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package domain
+
+import (
+	"fmt"
+	"github.com/knative/client-contrib/plugins/kn-admin/cmd"
+	"github.com/spf13/cobra"
+)
+
+// domainCmd represents the domain command
```

Comment:
I think you need to repeat the name of the function, e.g., “NewDomainCmd” as the first word in comment